### ISSUE TITLE
Bower package not found?

### ISSUE THREAD

**Issue by Jakobud** [BitPiston/semantic-ui-scss#1]
```
# bower install semantic-ui-scss
bower ENOTFOUND     Package semantic-ui-scss not found
```

Uh.... what?

**Comment by Einkoro** [comment]
The package was never registered. The port hasn't progressed beyond a WIP due to issues with inheritance and lack of support using variables in import statements. There is a conversation on the Semantic UI issues about it

**Comment by Jakobud** [comment]
Yeah I think they all got closed. Too bad there isn't a way to make this happen. Is the problem just trying to include theme partials using variable names?

**Comment by Einkoro** [comment]
Phone cut me off pasting the URL, thread is here:
https://github.com/Semantic-Org/Semantic-UI/issues/1147

Thats correct. I've looked at alternatives using a structure outlined in the readme and building a bunch of prefixed variables but its prone to error. Dropping theme support would be the only viable way to push out a sass port similar to how Foundation's SCSS bower package is structured.

**Comment by Einkoro** [comment]
Ah I see you are already on that thread. Do you think there would be any interest in a version minus the theme support?

**Comment by Jakobud** [comment]
I would use it. Or at least try it. I would think it would be easy to add in theme support but just without using variables in the import names. Just import additional theme-specific partials to override your default variable values. I'll look at it closer this week and see if it's feasible or not.

**Comment by Einkoro** [comment]
I was playing around with compiling the !default variables into each component partial like foundation and then you can load your own variables in a partial before the components. The problem I ran into was some global variables are redefined per component which would likely require changes upstream and talking to Jack changing them was a less than desirable approach.

My experimental solution to that was to create component specific versions of the globals and inherit the globals unless one was defined in the component. It made things a pain to override from your own variables partial in my experience so I dropped it at that point.
